modules/sdk-unified-wallet/src/unifiedWallets.ts

```diff
@@ -95,8 +95,12 @@ export abstract class UnifiedWallets implements IUnifiedWallets {
     return newWallet;
   }
 
-  async getAllUnifiedWallets(): Promise<UnifiedWallet[]> {
-    return await this.bitgo.get(this.bitgo.url(this.urlPath, 2)).result();
+  async getUnifiedWallets(
+    paginationOptions: PaginationOptions = { prevPage: 0, limit: 15, order: 'DESC' }
+  ): Promise<UnifiedWallet[]> {
+    const { limit, order, prevPage } = paginationOptions;
```

Comment:
nit: you can actually do `const { limit, order, prevPage: page } = paginationOptions;`

Reply:
thanks for the tip :) ... implemented